packages/sample-project/src/App.svelte

```diff
@@ -132,11 +113,15 @@
       },
     ],
   };
+
+  $: viewVL ? console.log("Vega-Lite view: ", viewVL.data("table")) : "";
```

Comment:
There is no Vega-Lite view. It's still a Vega view. Not sure you need to change this here since it's for different examples but just wanted to point it out.

Reply:
TIL